docs/Project.toml

```
@@ -4,4 +4,5 @@ JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 OffsetArrays = "6fe1bfb0-de20-5000-8ca7-80f57d26f881"
 
 [compat]
-Documenter = "0.25"
+Documenter = "0.25, 0.26"
```

I interpret it as "building the documentation using `Documenter@0.26`".

```suggestion
Documenter = "0.26"
```